code/game/objects/structures/tables_racks.dm

```diff
@@ -562,7 +562,7 @@
 /obj/structure/table/woodentable/attackby(obj/item/I as obj, mob/user as mob, params)
 
 	if (istype(I, /obj/item/stack/tile/grass))
-		del(I)
+		qdel(I)
 		new /obj/structure/table/woodentable/poker( src.loc )
 		del(src)
 		visible_message("<span class='notice'>[user] adds the grass to the wooden table</span>")

```

Comment:
So, wow... in this code, the table... creates a new poker table (which will qdel the table it's replacing in its New), **directly deletes** itself (causing this proc to end immediately), and then seems to think it can display a message **after deleting itself.**